src/api/request.ts

```
@@ -0,0 +1,72 @@
+export interface RequestConfig {
+  method: HTTPMethodType;
+  url: string;
+  query?: Record<string, string | number | boolean>;
+  body?: Record<string, unknown> | FormData;
+  headers?: Record<string, string>;
+}
+
+export const request = async <T>(config: RequestConfig): Promise<T> => {
+  const { method, url, query, body, headers } = config;
+
+  const requestConfig: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)